teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>Simple examples for <strong>the </strong>PubChem API</blockquote><blockquote>Determine <s>the querying </s><strong>a query</strong> compound</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>